packages/communication-react/CHANGELOG.md

```diff
@@ -1,9 +1,18 @@
 # Change Log - @azure/communication-react
 
-This log was last generated on Thu, 22 Jul 2021 17:42:41 GMT and should not be manually modified.
+This log was last generated on Mon, 16 Aug 2021 21:18:19 GMT and should not be manually modified.
 
 <!-- Start content -->
 
+## [1.0.0-beta.4](https://github.com/azure/communication-ui-library/tree/@azure/communication-react_v1.0.0-beta.4)
+
+Mon, 16 Aug 2021 21:18:19 GMT 
+[Compare changes](https://github.com/azure/communication-ui-library/compare/@azure/communication-react_v1.0.0-beta.3..@azure/communication-react_v1.0.0-beta.4)
+
+### Changes
```

Comment:
@alcail here we manually update the changelog, beachball sadly can't generate nice changelogs automatically since we switched to this meta package. So for other releases take a look at the changelog.mds of the other packages - they should have all the changes for this release - then add them here (can remove unnecessary ones, combine similar ones, or rewrite poorly worded changes - easiest way to know is by asking yourself - 'would this be useful to me as a developer')